src/Core/Style.js

```diff
@@ -0,0 +1,113 @@
+import { FEATURE_TYPES } from 'Core/Feature';
+
+function rgba2rgb(orig) {
+    if (orig && orig.includes('rgba')) {
+        const rgb = orig.replace(/\s/g, '').match(/^rgba?\((\d+),(\d+),(\d+),(\d\.\d)/i);
```

Comment:
You can drop the `rgba` match, as well as `hsla` below. Also opacity should be `[\d.]+`.